governance/bylaws.md

```diff
@@ -186,11 +189,46 @@ The time and place of every annual or special meeting shall be given to each mem
 All other resolutions or motions at an annual or special meeting of members of the Association shall be passed or rejected by a simple majority of the votes cast by Regular or Fellow Members.
-At the discretion of the Board, votes may be cast in person at the meeting, or received by the Secretary by mail or electronic transmission twenty-four (24) hours in advance of the meeting unless otherwise required by the Bylaws of the Association. 
+Votes may be cast by,
+1. indicating Yea/Nay, either in person, by proxy, telephonically or electronically
+2. a secret vote that is,
+   1. marked on a secret ballot cast personally, electronically or by a proxy
+
+##### 3.4.1	Absentee Voting
+Voting by mail or by telephonic or electronic means may only be used if,
+1. the votes may be verified as having been made by members entitled to vote; and
+2. the corporation is not able to identify how each member voted.
+
+#### 3.5	Proxies
+Every member entitled to vote at a meeting of the members may by means of a proxy appoint a proxyholder or one or more alternate proxyholders as the member’s nominee to attend and act at the meeting in the manner, to the extent and with the authority conferred by the proxy.
+
+##### 3.5.1	Who may be a proxyholder
+Only certified members may be appointed as proxies.
+
+##### 3.5.2	Form of Proxy
+A proxy form must be deposited to the meeting chair, by mail, physically or by electronic means, no later than 24 hours before the meeting.
+The proxy form,
+1. must be signed by the member (physically, digitally or cryptographically) and;
+2. must indicate what meeting the proxy applies to and;
+3. must indicate what voting actions are authorised.
+
+##### 3.5.3	Validity
+A proxy is valid only at the meeting for which it is given or, if that meeting is adjourned, at the meeting that constitutes the adjourned meeting.
+
+##### 3.5.4	Revocation
+A member may revoke a proxy
+1. by depositing with the meeting chair before the meeting, or, if that meeting is adjourned, before the beginning of the continued meeting, a revocation that is signed by the member
+
+##### 3.5.5	Proxy Voting by Yea/Nay
+If the chair of a meeting of the members declares to the meeting that, to the best of their belief, if a ballot is conducted, the total number of votes of members represented at the meeting by proxy required to be voted against a matter or group of matters to be decided at the meeting is less than 5 per cent of all the votes that might be cast at the meeting on such ballot, and if a member, proxyholder or alternate proxyholder does not demand a ballot,
+1. the chair may conduct the vote in respect of that matter or group of matters by voicing YEA/NAY and
+2. a proxyholder or alternate proxyholder may vote in respect of that matter or group of matters by voicing YEA/NAY
```

Comment:
This is an interesting problem for proxyholders.  If we vote audibly "Yea/Nay" how do we make sure that the volume is amplified by the number of votes being cast?  One of the reasons I propose to leave it with the ISP to manage is that person ought to be capable of calling for a secret ballot if they have many proxies to cast on a contentious issue.